lib/media/drm_engine.js

```diff
@@ -1443,6 +1481,33 @@ shaka.media.DrmEngine = class {
     this.playerInterface_.onKeyStatus(shaka.util.MapUtils.asObject(publicMap));
   }
 
+  /**
+   *
+   * @param {string} keySystem
+   *
+   * @private
+   */
+  static isPlayReadyKeySystem_(keySystem) {
+    return keySystem === 'com.microsoft.playready.recommendation'
+        || keySystem === 'com.microsoft.playready';
```

Comment:
nit: The `||` part goes on the previous line and we don't use `===`, just use `==`.